src/RadworksGovernor.sol

```diff
@@ -0,0 +1,229 @@
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "Radworks Governor Bravo";
+
+  /// @notice The number of RAD (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 100_000e18; // 100,000 RAD
+
+  /// @param _initialVotingDelay The initial voting delay this Governor will enforce.
+  /// @param _initialVotingPeriod The initial voting period this Governor will enforce.
+  /// @param _initialProposalThreshold The initial number of RAD required to submit
+  /// a proposal this Governor will enforce.
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(RAD_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    RadworksGovernorTimelockCompound(TIMELOCK)
+    Governor(GOVERNOR_NAME)
+  {}
+
+  ///
+  /// @dev Mapping from proposal ID to vote tallies for that proposal.
+  ///
+  mapping(uint256 => ProposalVote) private _proposalVotes;
+
+  ///
+  /// @dev Mapping from proposal ID and address to the weight the address
+  /// has cast on that proposal, e.g. _proposalVotersWeightCast[42][0xBEEF]
+  /// would tell you the number of votes that 0xBEEF has cast on proposal 42.
+  ///
+  mapping(uint256 => mapping(address => uint256)) private _proposalVotersWeightCast;
+
+  /// @inheritdoc Governor
+  function supportsInterface(bytes4 interfaceId)
+    public
+    view
+    virtual
+    override(Governor, RadworksGovernorTimelockCompound)
+    returns (bool)
+  {
+    return RadworksGovernorTimelockCompound.supportsInterface(interfaceId);
+  }
+
+  ///
+  /// @dev See {IGovernor-COUNTING_MODE}.
+  ///
+  // solhint-disable-next-line func-name-mixedcase
+  function COUNTING_MODE() public pure override returns (string memory) {
+    return "support=bravo&quorum=bravo";
+  }
+
+  ///
+  /// @dev See {IGovernor-hasVoted}.
+  ///
+  function hasVoted(uint256 proposalId, address account)
+    public
+    view
+    virtual
+    override
+    returns (bool)
+  {
+    return _proposalVotersWeightCast[proposalId][account] > 0;
+  }
+
+  ///
+  /// @dev See {Governor-_quorumReached}.
+  ///
+  function _quorumReached(uint256 proposalId) internal view virtual override returns (bool) {
+    ProposalVote storage proposalVote = _proposalVotes[proposalId];
+
+    return quorum(proposalSnapshot(proposalId)) <= proposalVote.forVotes + proposalVote.abstainVotes;
+  }
+
+  ///
+  /// @dev See {Governor-_voteSucceeded}. In this module, forVotes must be > againstVotes.
+  ///
+  function _voteSucceeded(uint256 proposalId) internal view virtual override returns (bool) {
+    ProposalVote storage proposalVote = _proposalVotes[proposalId];
+
+    return proposalVote.forVotes > proposalVote.againstVotes;
+  }
+
+  function _countVote(
```

Comment:
this is the only method without documentation; may want to add something

Reply:
Done b68221bc769e3077b565ceb3a8785624f038c455